trino/trino.go

```diff
@@ -1881,6 +1887,16 @@ func scanNullTime(v interface{}) (NullTime, error) {
 	if len(vparts) > 1 && !unicode.IsDigit(rune(vparts[len(vparts)-1][0])) {
 		return parseNullTimeWithLocation(vv)
 	}
+	// Time literals may not have spaces before the timezone.
+	if strings.ContainsRune(vv, '+') {
+		return parseNullTimeWithLocation(strings.Replace(vv, "+", " +", 1))
+	}
+	hiphenCount := strings.Count(vv, "-")
+	// We need to ensure we don't treat the hyphens in dates as the minus offset sign.
+	// So if there's only one hyphen or more than 2, we have a negative offset.
+	if hiphenCount == 1 || hiphenCount > 2 {
```

Comment:
nit: typo, also I'm not a native speaker and had to check if it's better to call this minus or hyphen, and TIL about [hyphen-minus](https://en.wikipedia.org/wiki/Hyphen-minus) 😄  
```suggestion
	hyphenCount := strings.Count(vv, "-")
	// We need to ensure we don't treat the hyphens in dates as the minus offset sign.
	// So if there's only one hyphen or more than 2, we have a negative offset.
	if hyphenCount == 1 || hyphenCount > 2 {
```